doc/src/conf.py

```diff
@@ -372,7 +372,7 @@
 
 \usepackage{bm}
```

Comment:
This serves nothing because there is not a single usage of `\bm` in math mark-up in sources.  But who knows it may help in future. (MathJax in its default configuration as set-up by current Sphinx appears to render the `\bm`; of course the mark-up has primarily to work for HTML output so if `\bm{}` syntax was not understood by HTML this would be a no-go anyhow).

Reply:
This seems to be a holdover from galgebra, which was removed from sympy (see 8a29065982af9e8c512824688ded9b41eacbff56). We can remove it if it is unused.

Reply:
I pushed a commit at PR #26868 which removes `\usepackage{bm}`.